appsdk/sdk.go

```diff
@@ -345,6 +354,11 @@ func (sdk *AppFunctionsSDK) Initialize() error {
 			loggerInitialized = true
 		}
 
+		// Verify that Core Services major version matches this SDK's major version
```

Comment:
if i set the ignoreVersionError to true, i would think the call wouldnt happen at all? skip this entirely

Reply:
I choose to still do the check so any errors are still be logged so user is aware. Thus why I choose ignoreVersionError rather than skipVersionCheck.